engine/src/test/scala/cromwell/engine/workflow/lifecycle/execution/ejea/EjeaBackendIsCopyingCachedOutputsSpec.scala

```diff
@@ -112,7 +112,7 @@ class EjeaBackendIsCopyingCachedOutputsSpec extends EngineJobExecutionActorSpec
           s"not invalidate a call for caching if backend coping failed when invalidation is disabled, when it was going to receive $hashComboName, if call caching is $mode" in {
             val invalidationDisabledOptions = CallCachingOptions(invalidateBadCacheResults = false)
             val cacheInvalidationDisabledMode = mode match {
-              case CallCachingActivity(rw, _) => CallCachingActivity(rw, invalidationDisabledOptions)
+              case CallCachingActivity(rw, options) => CallCachingActivity(rw, invalidationDisabledOptions)
```

Comment:
Hmm because I probably changed to `_` in the first version and then git reverted it to what it was :) Will change it